tgui/packages/tgui/interfaces/AresAdmin.js

```diff
@@ -381,17 +393,39 @@ const MainMenu = (props, context) => {
           </Stack.Item>
           <Stack.Item>
             <Button
-              content="Remote Access Log"
-              tooltip="View which admins have been using ARES."
-              icon="user-secret"
+              content="Bioscan"
+              icon="satellite"
               ml="auto"
               px="2rem"
               width="25vw"
               bold
-              onClick={() => act('page_admin_list')}
+              onClick={() => act('bioscan')}
+              disabled={remotelock}
+              tooltip="Trigger an immediate bioscan for diagnostics."
             />
           </Stack.Item>
         </Stack>
+      </Section>
+      <Section>
+        {remotelock && (
+          <Stack>
+            <Stack.Item grow>
+              <h3>Remote Admin</h3>
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                content="Remote Access Log"
```

Comment:
Just a heads up that after https://github.com/cmss13-devs/cmss13/pull/6244 buttons should not use the content property. Instead their content needs to be as a child e.g.

```js
<Button content="Bioscan" />
```

Needs to instead be:
```js
<Button>Bioscan</Button>
```